<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>675</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>380</y>
      <width>804</width>
      <height>192</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="4" column="0">
      <widget class="QLineEdit" name="Add_ValueEdit">
       <property name="maxLength">
        <number>9</number>
       </property>
       <property name="placeholderText">
        <string>Значение</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="Add_KeyEdit">
       <property name="maxLength">
        <number>9</number>
       </property>
       <property name="placeholderText">
        <string>Ключ</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="RemoveButton">
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="SearchButton">
       <property name="text">
        <string>Искать</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="SearchEdit">
       <property name="maxLength">
        <number>9</number>
       </property>
       <property name="placeholderText">
        <string>Ключ</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="AddButton">
       <property name="text">
        <string>Добавить </string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="RemoveEdit">
       <property name="maxLength">
        <number>9</number>
       </property>
       <property name="placeholderText">
        <string>Ключ</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QPushButton" name="Create2TablesButton">
       <property name="text">
        <string>Сделать 2 таблицы с ключами разных знаков</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QRadioButton" name="LeftTableButton">
       <property name="text">
        <string>Левая таблица</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QRadioButton" name="RightTableButton">
       <property name="text">
        <string>Правая таблица</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>961</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableWidget" name="tableWidget_1">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <column>
        <property name="text">
         <string>Ключ</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Значение</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget_2">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <column>
        <property name="text">
         <string>Ключ</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Значение</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1023</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
